ChatQnA/docker_compose/intel/hpu/gaudi/compose.yaml

```diff
@@ -59,6 +59,7 @@ services:
       https_proxy: ${https_proxy}
       REDIS_URL: ${REDIS_URL}
       INDEX_NAME: ${INDEX_NAME}
+      TEI_EMBEDDING_ENDPOINT: ${TEI_EMBEDDING_ENDPOINT}
```

Comment:
hi, the environment variable should be TEI_ENDPOINT? refer this https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/redis/langchain/prepare_doc_redis.py#L37

Reply:
https://gitee.com/bigclouds99/GenAIExamples/tree/1.1_with_wrapper/ChatQnA/docker_compose/intel/hpu/gaudi#2-build-retriever-image

https://github.com/opea-project/GenAIComps/blob/main/comps/retrievers/redis/langchain/retriever_redis.py#L34

Reply:
Sorry, I got the wrong line number